src/__tests__/lib/exampleData.js

```diff
@@ -595,7 +595,7 @@ export const action = Object.freeze({
     apiKey: selfAccount.apiKey,
   }): LoginSuccessAction),
   realm_init: (deepFreeze({
-    type: REALM_INIT,
+    type: REGISTER_COMPLETE,
```

Comment:
Should also update the name of the example action itself.

(I found this with the following search:
```
$ rg -i 'realm.?init'
```
looking for any variation of the name with or without caps or underscores.)